packages/dante3/src/popovers/menuBar.tsx

```diff
@@ -128,7 +136,7 @@ export default function MenuBar({ editor, fixed }) {
 
     return (
       <AnchorStyle
-        fixed={fixed}
+        //fixed={fixed}  // This wasn't present in the type for AnchorStyle and didn't appear to be referenced anywhere
```

Comment:
lets keep this for now

Reply:
I might be able to add fixed as a prop to AnchorStyle, but I'm still trying to figure out how to work with the types on those styled components